### PR TITLE
update ReadMe for configurations of iOS Safari run

### PR DESCRIPTION
**Pull request by hemano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ Add the below dependencies in your pom.xml (Master)
 
 ## [Tips](https://github.com/saikrishna321/AppiumTestDistribution/wiki/Tips)
 
+## [iOS Safari run config](https://github.com/hemano/AppiumTestDistribution/wiki/Configurations-for-iOS-Safari-browser-test-run)
```
Comment (SrinivasanTarget) [inline README.md]: I would rather request you to raise a PR for Wiki as well here. Good to have the wiki updated here : https://github.com/saikrishna321/AppiumTestDistribution/wiki

Comment (saikrishna321) [reply README.md]: @hemano i agree with Srini and the issue with simulator parallel for web test is coz of apple bug

Refer: https://github.com/appium/appium/issues/9209

You need to remove the Safari Simulator capabilities section

Comment (hemano) [reply README.md]: @SrinivasanTarget It seems Git doesn't support PR for Wiki and I don't have access to this repo to create a new wiki page. Could you help in posting it on wiki
@saikrishna321 I ran tests on a single simulator and they worked fine. Also, I've clearly mentioned about the bug you referred. Should I remove the simulator section?

Comment (SrinivasanTarget) [reply README.md]: Can you try this https://github.com/saikrishna321/AppiumTestDistribution.wiki.git

Comment (hemano) [reply README.md]: I cloned it, created a new branch, added a new page, committed and pushed. Access denied 403.

**Pull request closed without merging**